src/styles/util.scss

```diff
@@ -89,3 +89,11 @@
 .u-color-text--muted {
   color: var.$color-text--light;
 }
+
+// TODO: The sidebar should really apply vertical spacing to its immediate children,
+// but until that happens, this utility class allows a container to assert
```

Comment:
Note that actually doing this (making the main sidebar layout element apply margin for its children) has proven oddly challenging in the past, or I would have taken care of it in this PR.

Reply:
> Note that actually doing this (making the main sidebar layout element apply margin for its children) has proven oddly challenging in the past,

Which element do you mean by "main sidebar layout element"?

Reply:
You're right. I was getting tired and didn't due my due commenting diligence here. The layout "expectation" is that elements in the sidebar are vertically spaced evenly. I can't define "sidebar" better yet because the document structure for the sidebar isn't the clearest. 

I started outlining the structure of the sidebar elements that are in play here...but it quickly became complex enough that I’m going to punt on the specifics: there’s another task here, which is to re-examine the markup of the sidebar and consolidate it, in terms of DOM structure. e.g. We attach `SidebarPanel`s to different elements than other message components, etc. etc.

The ideal, IMO, would be if there were a main “sidebar” containing element that had as its immediate children (not an exhaustive list):

* The tabs
* Any panels or messages
* A container for all of the threads

Then we could apply a vertical-spacing rule to the container and things would lay out evenly (we’d also apply vertical-spacing to the container with the threads in it, instead of applying bottom margin to each `ThreadCard`, see?).

The overarching convention here is that margins (spacing) is the duty of the parent element, whereas the complexity of the sidebar markup has necessitated managing margins on an element itself, which is fiddly, more repeated styles, and slightly fragile. Yeah, we should fix this! But I didn’t think I could bite it off in this PR. The intent here was to centralize it in a utility class so that we can find it and zap it when we fix this for real.

So, I can’t give you a “main sidebar element” because none exists, but I can certainly improve this comment. The goal is to _have_ a main sidebar content container element.